src/app/search/state/search.service.ts

```diff
@@ -534,7 +534,7 @@ export class SearchService {
       ['verified_platforms.keyword', 'Verified Platforms'],
       ['categories.name.keyword', 'Category'],
       ['descriptor_type_versions.keyword', 'Language Versions'],
-      ['openData', 'Open Data'],
+      ['openData', 'Open'],
```

Comment:
I think you're being overly careful here, but as in the SEAB ticket, I think it's fine to consider this "Open Data" due to expectations in this field

Reply:
Yeah I prefer "Open Data" to "Open", more clear to me at least. But if it's changed in the facet I think it would make sense for it to match with its header in the versions table... as Denis mentioned we may need two rows in a future ticket